client/src/components/Sort/SortOptionsList.jsx

```diff
@@ -1,14 +1,23 @@
+// @flow
+
 import inlineStyles from "../../static/inline.css";
 import marginLeftStyles from "../../static/margin-left.css";
 
 import React from "react";
-import PropTypes from "prop-types";
 import SortOption from "./SortOption";
 import SORT_TYPE from "./sortType";
 
-const SortOptionsList = ({ currentSortType, onSwitchSortType }) => {
-  const sortOptions = Object.values(SORT_TYPE).map(sortType => (
-    <SortOption
+type SortOptionsListProps = {
+  currentSortType: string,
+  onSwitchSortType: function
+};
+
+const SortOptionsList = (sortOptionListProps:SortOptionsListProps) => {
+  const { currentSortType, onSwitchSortType } = sortOptionListProps;
+
+  const sortOptions = Object.values(SORT_TYPE).map((sortType:mixed) => (
```

Comment:
_minor_

As long as you are in full of control over the code you've been written, it would be better to explicitly signify types that you're using, here, `sortType` is definitely is a `string` type.

P.S. Using types `mixed`, `any` etc will significantly impair type checking policy in the vast majority of cases and tools you're using(flow, typescript) and in some cases simply would mean that type checking is turned off for this particular piece of code. These type of annotations should be used sparingly in the prod codebase.